modulefiles/tasks/wcoss2/nexus_gfs_sfc.local.lua

```diff
@@ -0,0 +1 @@
+load("python_srw")
```

Comment:
If this is the only thing it does, that is no longer necessary. This one should be loaded by default.

Reply:
Removed.